### PR TITLE
Minor cleanups.

### PR DESCRIPTION
**Pull request by bangerth**
Found while looking at #9795.
/rebuild

### CODE REVIEW

File: source/grid/grid_in.cc [hunk 1]
```diff
@@ -411,10 +407,10 @@ GridIn<dim, spacedim>::read_vtk(std::istream &in)
                         {
                           double id;
                           in >> id;
-                          if (set == "MaterialID")
+                          if (keyword == "MaterialID")
```
Comment (masterleinad) [inline source/grid/grid_in.cc:410]: AFAICT, these have different values. `set` is the next known data set name after `SCALARS` while we are still updating `keyword`.

Comment (bangerth) [reply source/grid/grid_in.cc:410]: No, look at line 374-379 (old) above: We do `set = keyword` if we find something (and `continue` if we don't). I think this really is correct.

Comment (masterleinad) [reply source/grid/grid_in.cc:410]: Yes, but we change `keyword` in lines 389 and 395. I believe this is responsible for the failing tests.

**Comment by bangerth** [comment]
@masterleinad -- You're totally right, of course. I've just given the field name a more descriptive variable name -- the root cause for me missing this is because we keep reusing the variable `keyword` for things that are not actually keywords.

This now succeeds with all relevant tests on my machine.

**Pull request merged**